redash/query_runner/script.py

```diff
@@ -51,16 +54,14 @@ def run_query(self, query, user):
             json_data = None
             error = None
 
-            query = query.strip()
+            script = query
 
             if self.configuration["path"] != "*":
                 script = os.path.join(self.configuration["path"], query.split(" ")[0])
                 if not os.path.exists(script):
                     return None, "Script '%s' not found in script directory" % query
 
                 script = os.path.join(self.configuration["path"], query).split(" ")
-            else:
-                script = query.split("*/ ")[1]
```

Comment:
Why remove this?

Reply:
@arikfr I think this code wrote for remove annotation from query.

Currently, `annotate_query` returns False. its query doesn't have an annotation.

Reply:
No, this is part of the original script query runner implementation. I'm not sure what exactly it's doing though :)

Reply:
Haha, I guess this code block want to take script name without annotation.

But I'm not sure too 🤔 

In this PR, I set on line 57 `script = query` instead of `script = query.split("*/ ")[1]`.

Do you have any good ideas?